lib/model/cardinalPortPosition.js

```diff
@@ -15,6 +17,7 @@ var CardinalPortPosition = function(direction,percentage) {
  * @function getDirection
  */
 CardinalPortPosition.prototype.getDirection = function() {
+  "use strict";
```

Comment:
I'd much rather have it be part of the build process. Unless there are many times I'd *not* want to `use strict` I'd rather not think about it and write it everywhere.